src/bundle/Resources/public/js/scripts/fieldType/ezauthor.js

```diff
@@ -118,6 +118,12 @@
 
             bulkDeleteButton.setAttribute('disabled', 'disabled');
 
+            const authorRowsExist = !!container.querySelector(SELECTOR_AUTHOR);
```

Comment:
Not sure but maybe:
```suggestion
            const authorsRowsExist = !!container.querySelector(SELECTOR_AUTHOR);
```

Reply:
This checks if at least one author row exists so `authorRowExists`?